content/en/docs/contributing/release-process.md

```diff
@@ -15,28 +15,38 @@ perform a cert-manager release:
 
 1. Install the [`release-notes`](https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md) CLI:
    ```sh
-   (cd && GO111MODULE=on go get k8s.io/release/cmd/release-notes@latest)
+   (cd && GO111MODULE=on go get k8s.io/release/cmd/release-notes@v0.7.0)
    ```
 2. Install our [`cmrel`](https://github.com/cert-manager/release) CLI:
```

Comment:
This should be clone the cmrel repo as everything needs to be run from within the checked-out repository

Reply:
I added a new line with the warning "remember to stay in the cloned `release/` folder while typing all the commands",m what do you think?